server/java/README.md

```diff
@@ -4,8 +4,9 @@ The TicketHub sample app server component.
 
 ## How to run
 
-1. `make run`.
-2. The server opens a listener on `0.0.0.0:4000`.
+1. Compile the [opa-java](https://github.com/StyraInc/opa-java) SDK and place the resulting JAR in `./libs`
```

Comment:
Do we still need this? The [build.gradle line here](https://github.com/StyraInc/styra-demo-tickethub/pull/90/files#diff-627c016d42c8d087da6d6540b08b557798e301f5f7db6736824b6d14f5defd03R42) makes me think that it would be pulled in by gradle.

Reply:
ccdb60a